packages/plugin-core/src/test/suite-integ/MoveNoteCommand.test.ts

```diff
@@ -394,53 +394,20 @@ suite("MoveNoteCommand", function () {
             })
           )
         ).toBeFalsy();
-        done();
-      },
-    });
-  });
-
-  // TODO: this test is flaky
-  test.skip("replace existing note", (done) => {
-    runLegacyMultiWorkspaceTest({
-      ctx,
-      preSetupHook: async ({ wsRoot, vaults }) => {
-        ENGINE_HOOKS.setupBasic({ wsRoot, vaults });
-      },
-      onInit: async ({ engine, vaults, wsRoot }) => {
-        const notes = engine.notes;
-        const vaultFrom = vaults[0];
-        const vaultTo = vaults[0];
-        const fooNote = NoteUtils.getNoteByFnameV5({
-          fname: "foo",
-          notes,
-          vault: vaultFrom,
-          wsRoot,
-        }) as NoteProps;
-        await WSUtils.openNote(fooNote);
-        const cmd = new MoveNoteCommand();
-        HistoryService.instance().subscribev2("lookupProvider", {
-          id: "move",
-          listener: async (event) => {
-            expect(event.action).toEqual("error");
-            done();
-          },
-        });
-        await cmd.run({
-          nonInteractive: true,
-          initialValue: "bar",
-          vaultName: vaultTo.fsPath,
-        });
-      },
-    });
-  });
+      });
+    }
+  );
 
-  test("move scratch note ", (done) => {
-    runLegacyMultiWorkspaceTest({
-      ctx,
+  describeMultiWS(
+    "WHEN move scratch note",
+    {
       preSetupHook: async ({ wsRoot, vaults }) => {
         await ENGINE_HOOKS.setupBasic({ wsRoot, vaults });
       },
-      onInit: async ({ engine, vaults, wsRoot }) => {
+    },
+    () => {
+      test("THEN do right thing", async () => {
```

Comment:
THEN do right thing? 😛

Reply:
time boxed session for fixing tests, figured fixing the logic was better than not 😅